compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp

```diff
@@ -100,10 +100,10 @@ static Value truncateFloat(OpBuilder &builder, Location loc, AffineMap inputMap,
         // used by attention's exp2 who's value is always > 0.
         Value mx = builder.create<arith::ConstantOp>(
             loc, builder.getFloatAttr(srcTy, mxDbl));
-        Value clamped = b.create<arith::MinimumFOp>(loc, mx, args[0]);
+        Value sel0 = b.create<arith::MinimumFOp>(loc, mx, args[0]);
```

Comment:
NIT: Can we change the name back to `clamped`, `sel0` was the old name because it was clamping with `arith::selectOp`.

Reply:
Noted